frontend/app/src/scenes/Home/index.tsx

```diff
@@ -37,17 +38,27 @@ class Home extends React.Component<HomeProps> {
               </div>
             </Link>
           </div>
-          <Menu theme="dark" mode="inline" defaultSelectedKeys={['1']}>
+          <Menu theme="dark" mode="inline" defaultOpenKeys={['sub1']}>
             <Menu.Item key="1" icon={<AreaChartOutlined />}>
               <Link to="/dashboard/home/" className={styles.txtWhite}>
                 Dashboard
               </Link>
             </Menu.Item>
-            <Menu.Item key="2" icon={<BankOutlined />}>
-              <Link to="/dashboard/universities/" className={styles.txtWhite}>
-                Universities
-              </Link>
-            </Menu.Item>
+            <Menu.SubMenu
+              key="sub1"
```

Comment:
Let's add meaningful & unique key names to this menu. 

Ex: "dashboard", "multiverse", "multiverse", "multiverse_universities"

Reply:
ok